File: Practical/HTTP_Example/NodeJS_Clients/PUT.js
const axios = require('axios')

// Program to make a simple GET request and parse the Response Body
const HOST = "http://localhost:";
const PORT = "3000"
const PATH = "/sensor/value"

const REQUEST_BODY = {
    index: 0,
    value:34
}
// Make a GET request to Server
axios.put(HOST + PORT + PATH, REQUEST_BODY)
    .then((response)=>{
        const RESPONSE_BODY  = JSON.stringify(response.data);
        console.log("Data: "+RESPONSE_BODY+". Response Code: "+ response.status)
    }).catch(error=>{
        console.log(error);
    })

File: Practical/HTTP_Example/Python_Clients/PUT.py
# Program to perform simple GET request to HTTP server
import requests
import json

HOST = "http://localhost:"
PORT = "3000"
PATH = "/sensor/value"
REQUEST_BODY = {
    "index":1,
    "value":1214
}

response = requests.put(HOST + PORT + PATH, REQUEST_BODY)

print(response.json())